### ISSUE TITLE
Having Problems Installing Packeges

### ISSUE THREAD

**Issue by michaelkali** [Und3rf10w/kali-anonsurf#17]
hi
im having a issues with installing anonsurf
i already re-installed kali twice and im still facing the same issue
heres the code:
~/kali-anonsurf# bash ./installer.sh
--2016-11-28 15:19:01--  https://geti2p.net/_static/i2p-debian-repo.key.asc
Resolving geti2p.net (geti2p.net)... 91.143.92.136, 2a02:180:a:65:2456:6542:1101:1010
Connecting to geti2p.net (geti2p.net)|91.143.92.136|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 14455 (14K) [text/plain]
Saving to: ‘/tmp/i2p-debian-repo.key.asc’

/tmp/i2p-debian-rep 100%[===================>]  14.12K  --.-KB/s    in 0.1s    

2016-11-28 15:19:02 (123 KB/s) - ‘/tmp/i2p-debian-repo.key.asc’ saved [14455/14455]

OK
Get:2 http://deb.i2p2.no unstable InRelease [12.1 kB]
Get:3 http://deb.i2p2.no unstable/main amd64 Packages [3,224 B]         
Hit:1 http://kali.mirror.garr.it/mirrors/kali kali-rolling InRelease
Fetched 15.4 kB in 1s (10.0 kB/s)                   
Reading package lists... Done
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  killyourtv-keyring
The following NEW packages will be installed:
  i2p-keyring killyourtv-keyring
0 upgraded, 2 newly installed, 0 to remove and 1333 not upgraded.
Need to get 25.3 kB of archives.
After this operation, 71.7 kB of additional disk space will be used.
Get:1 http://deb.i2p2.no unstable/main amd64 i2p-keyring all 2016.10.01 [12.4 kB]
Get:2 http://deb.i2p2.no unstable/main amd64 killyourtv-keyring all 2015.03.27.1 [12.9 kB]
Fetched 25.3 kB in 0s (53.4 kB/s)            
Selecting previously unselected package i2p-keyring.
(Reading database ... 303568 files and directories currently installed.)
Preparing to unpack .../0-i2p-keyring_2016.10.01_all.deb ...
Unpacking i2p-keyring (2016.10.01) ...
Selecting previously unselected package killyourtv-keyring.
Preparing to unpack .../1-killyourtv-keyring_2015.03.27.1_all.deb ...
Unpacking killyourtv-keyring (2015.03.27.1) ...
Setting up i2p-keyring (2016.10.01) ...
Adding/refreshing keys:
  67ECE5605BCF1346 2013-10-10 I2P Debian Package Repository <killyourtv@i2pmail.org> [ expires 2017-10-01 ]
... Warning: apt-key should not be used in scripts (called from postinst maintainerscript of the package i2p-keyring)
OK
Setting up killyourtv-keyring (2015.03.27.1) ...
Adding/refreshing keys:
  8CF6E896B3C01B09 2013-04-06 KillYourTV Debian repository signing key <killyourtv@mail.i2p> [ expires 2016-04-10 ]
... Warning: apt-key should not be used in scripts (called from postinst maintainerscript of the package killyourtv-keyring)
OK
Removing old keys (if exist):
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 i2p : Depends: i2p-router (>= 0.8.6-5) but it is not going to be installed
       Depends: libjbigi-jni but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
dpkg-deb: building package 'kali-anonsurf' in 'kali-anonsurf.deb'.
Selecting previously unselected package kali-anonsurf.
(Reading database ... 303578 files and directories currently installed.)
Preparing to unpack kali-anonsurf.deb ...
Unpacking kali-anonsurf (1.2.2) ...
dpkg: dependency problems prevent configuration of kali-anonsurf:
 kali-anonsurf depends on secure-delete; however:
  Package secure-delete is not installed.
 kali-anonsurf depends on tor; however:
  Package tor is not installed.
 kali-anonsurf depends on i2p; however:
  Package i2p is not installed.

dpkg: error processing package kali-anonsurf (--install):
 dependency problems - leaving unconfigured
Processing triggers for systemd (231-4) ...
Errors were encountered while processing:
 kali-anonsurf
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Correcting dependencies... Done
The following packages were automatically installed and are no longer required:
  libservice-wrapper-java libservice-wrapper-jni secure-delete service-wrapper
  tor tor-geoipdb torsocks
Use 'apt autoremove' to remove them.
The following additional packages will be installed:
  secure-delete tor tor-geoipdb torsocks
Suggested packages:
  mixmaster torbrowser-launcher tor-arm apparmor-utils obfsproxy obfs4proxy
The following packages will be REMOVED:
  kali-anonsurf
The following NEW packages will be installed:
  secure-delete tor tor-geoipdb torsocks
0 upgraded, 4 newly installed, 1 to remove and 1333 not upgraded.
1 not fully installed or removed.
Need to get 2,569 kB of archives.
After this operation, 10.0 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://kali.mirror.garr.it/mirrors/kali kali-rolling/main amd64 secure-delete amd64 3.1-6 [69.6 kB]
Get:2 http://kali.mirror.garr.it/mirrors/kali kali-rolling/main amd64 tor amd64 0.2.8.9-1 [1,399 kB]
Get:3 http://kali.mirror.garr.it/mirrors/kali kali-rolling/main amd64 torsocks amd64 2.2.0-1 [72.5 kB]
Get:4 http://kali.mirror.garr.it/mirrors/kali kali-rolling/main amd64 tor-geoipdb all 0.2.8.9-1 [1,028 kB]
Fetched 2,569 kB in 4s (563 kB/s)      
(Reading database ... 303587 files and directories currently installed.)
Removing kali-anonsurf (1.2.2) ...
Selecting previously unselected package secure-delete.
(Reading database ... 303578 files and directories currently installed.)
Preparing to unpack .../0-secure-delete_3.1-6_amd64.deb ...
Unpacking secure-delete (3.1-6) ...
Selecting previously unselected package tor.
Preparing to unpack .../1-tor_0.2.8.9-1_amd64.deb ...
Unpacking tor (0.2.8.9-1) ...
Selecting previously unselected package torsocks.
Preparing to unpack .../2-torsocks_2.2.0-1_amd64.deb ...
Unpacking torsocks (2.2.0-1) ...
Selecting previously unselected package tor-geoipdb.
Preparing to unpack .../3-tor-geoipdb_0.2.8.9-1_all.deb ...
Unpacking tor-geoipdb (0.2.8.9-1) ...
Setting up tor (0.2.8.9-1) ...
Something or somebody made /var/lib/tor disappear.
Creating one for you again.
Something or somebody made /var/log/tor disappear.
Creating one for you again.
update-rc.d: We have no instructions for the tor init script.
update-rc.d: It looks like a network service, we disable it.
insserv: warning: current start runlevel(s) (empty) of script `tor' overrides LSB defaults (2 3 4 5).
insserv: warning: current stop runlevel(s) (0 1 2 3 4 5 6) of script `tor' overrides LSB defaults (0 1 6).
Setting up secure-delete (3.1-6) ...
Setting up tor-geoipdb (0.2.8.9-1) ...
Processing triggers for systemd (231-4) ...
Processing triggers for man-db (2.7.5-1) ...
Setting up torsocks (2.2.0-1) ...
Selecting previously unselected package kali-anonsurf.
(Reading database ... 303660 files and directories currently installed.)
Preparing to unpack kali-anonsurf.deb ...
Unpacking kali-anonsurf (1.2.2) ...
dpkg: dependency problems prevent configuration of kali-anonsurf:
 kali-anonsurf depends on i2p; however:
  Package i2p is not installed.

dpkg: error processing package kali-anonsurf (--install):
 dependency problems - leaving unconfigured
Processing triggers for systemd (231-4) ...
Errors were encountered while processing:
 kali-anonsurf
please help!!
thanks

**Comment by Und3rf10w** [comment]
Duplicate of #13

**Comment by Und3rf10w** [comment]
Please pull down the latest commit in the ```master``` branch (704c453). I just pushed a bugfix for this issue. Something happened with Kali where it's not pulling down ```libjetty8-java```. Once you pull down that commit, attempt to rerun the installer.

**Comment by michaelkali** [comment]
thanks a lot it downloaded
